### ISSUE TITLE
AIR 926 seems to ignore orientation settings.

### ISSUE THREAD

**Issue by KupoGames** [airsdk/Adobe-Runtime-Support#2042]
When running on the latest version of AIR, my app seems to ignore these orientation settings:
```
<aspectRatio>landscape</aspectRatio>
<autoOrients>true</autoOrients>
```

The app just rotates without restriction, where as before it was locked to only the 2 landscape orientations.
Setting autoOrients to false does not seem to make a difference.

I read that AIR changed something related to orientation in the latest version - is there some code I should add to keep the old behaviour?

Thanks.

**Comment by ajwfrost** [comment]
Hi
Interesting: yes there was a change to this but we had checked that it only impacted the use of 'ignored' for that value .. but let me double-check!
thanks

**Comment by a9848840** [comment]
Yes
I have the same problem

`<aspectRatio>landscape</aspectRatio>`

currently does not work

**Comment by ajwfrost** [comment]
FYI we've been able to reproduce this too now, but cannot see how the failure was introduced -> working back and debugging, we may look to do a minor patch to get this part fixed...

thanks

**Comment by itlancer** [comment]
@ajwfrost, I found more issues around it https://github.com/airsdk/Adobe-Runtime-Support/issues/2047

**Comment by ajwfrost** [comment]
Thanks @itlancer - yes we've just been looking at this. There's some error handling that had kicked in as part of the earlier change, in some conditions, but the handler was at too high a level and means that other code got skipped...

So currently we have a fix for the initial issue as outlined above, I'll ask the guys to check with the additional cases that you've highlighted in the test file in #2047...

thanks

**Comment by ajwfrost** [comment]
FYI we have a fix for this one now in the updated release: https://airsdk.harman.com/download

**Comment by KupoGames** [comment]
Thanks, I can confirm that orientation is working correctly again.

**Comment by ajwfrost** [comment]
Thanks (fixed in 33.1.1.929)
